articles/connect-end-user-to-wifi-with-certificate.md

```diff
@@ -0,0 +1,342 @@
+# Connect end users to Wi-Fi or VPN with a certificate (from DigiCert, NDES, or custom SCEP)
+
+_Available in Fleet Premium_
+
+Fleet can help your end users connect to Wi-Fi or VPN by adding your certificate authority to issue
+certificates. Fleet currently supports
+[DigiCert](https://www.digicert.com/digicert-one), [Microsoft
+NDES](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/network-device-enrollment-service-overview),
+and custom [SCEP](https://en.wikipedia.org/wiki/Simple_Certificate_Enrollment_Protocol) server.
+
+This guide will walk you through configuring your certificate authority and delivering certificates.
+
+## DigiCert
+
+To install certificates from DigiCert to hosts, do the following steps:
+
+- [Create service user in DigiCert](#step-1-create-service-user-in-digicert)
+- [Create certificate profile in DigiCert](#step-2-create-certificate-profile-in-digicert)
+- [Connect Fleet to DigiCert](#step-3-connect-fleet-to-digicert)
+- [Add a PKCS12 configuration profile to Fleet](#step-4-add-pkcs-12-configuration-profile-to-fleet)
```

Comment:
```suggestion
- [Add configuration profile to Fleet](#step-4-add-pkcs-12-configuration-profile-to-fleet)
```

I think we can cut the `PKCS12` to make this a little bit easier to read/digest. Less words/concepts.